lib/rtsp.c

```diff
@@ -165,6 +169,23 @@ bool Curl_rtsp_connisdead(struct connectdata *check)
   return ret_val;
 }
 
+/*
+ * Function to check on various aspects of a connection.
+ */
+static unsigned int rtsp_conncheck(struct connectdata *check,
+                                   unsigned int checks_to_perform)
```

Comment:
No, if you have immediate plans to actually use the parameter in your next PR then I don't think you need to remove it.